src/Cdn/Cdn/help/Az.Cdn.md

```diff
@@ -2,7 +2,7 @@
 Module Name: Az.Cdn
 Module Guid: 91832aaa-dc11-4583-8239-bce5fd531604
 Download Help Link: https://docs.microsoft.com/en-us/powershell/module/az.cdn
-Help Version: 4.2.2.0
+Help Version: 4.2.4.0
```

Comment:
why do you want to update version?

Reply:
This is file is auto updated. This could be the sdk version number I think. I have updated the sdk version from 4.2.2 to 4.2.4